src/main/cpp/SwerveModule.cpp

```diff
@@ -20,17 +20,14 @@ bool SwerveModule::Init(Location loc) {
 
     OKC_CHECK(this->steer_pid_ != nullptr);
     steer_pid_->EnableContinuousInput(0, 360);
+    steer_pid_->SetTolerance(2, 2); // tolerate 2 degrees of deviation, which shouldn't be a lot I don't think
```

Comment:
This should be added to that parameterization ticket

Reply:
#45